### PR TITLE
Multiple fixes and minor cleanup

### PR DESCRIPTION
**Pull request by Lethalinjectionx**
Cryo changes
Saddle changes
Removed noglin code until it can be reworked
Config updates
Fixed database error for sqlite

### CODE REVIEW

File: ArkShop/ArkShop/Private/Helpers.h [hunk 1]
```diff
@@ -45,12 +45,18 @@ namespace ArkShop
...
 
-	FORCEINLINE TArray<FString> GetSaddleData(APrimalDinoCharacter* dino)
+	FORCEINLINE TArray<FString> GetSaddleData(UPrimalItem* saddle)
 	{
-		TArray<FString> data{ "", "", "" };
-		TArray<FString> colors{ "0", "0", "0" };
+		TArray<FString> data;
+		TArray<FString> colors;
+
+		data.Add("");
```
Comment (SubstituteR) [inline ArkShop/ArkShop/Private/Helpers.h:53]: What's going on here?

File: ArkShop/ArkShop/Private/Helpers.h [hunk 2]
```diff
@@ -99,7 +106,16 @@ namespace ArkShop
 		strings.Add(tmp);
 		strings.Add(""); // empty
 		strings.Add("0"); // should get bitmasks for buffs but doesn't seem to get used
-		strings.Append(GetSaddleData(dino));
+
+		//strings.Append(GetSaddleData(saddle)); 
+		//bypassing saddle data for now
+		strings.Add("");
```
Comment (SubstituteR) [inline ArkShop/ArkShop/Private/Helpers.h:112]: What's going on here?

File: ArkShop/ArkShop/Private/Helpers.h (hunk 1 above)
Comment (Lethalinjectionx) [reply ArkShop/ArkShop/Private/Helpers.h:53]: Was getting a random crash when trying to pull information to display saddle information. Would rather get a console crash fix released as this is a minor display issue on a purchased dino with a saddle.

File: ArkShop/ArkShop/Private/Helpers.h (hunk 2 above)
Comment (Lethalinjectionx) [reply ArkShop/ArkShop/Private/Helpers.h:112]: Putting default values so the `customitemdata` is structured properly.

**Pull request merged**